backend/src/historicData/dto/cancellationStat.dto.ts

```diff
@@ -18,10 +18,23 @@ export class CancellationStat extends MaybeStats {
   })
   cancellationProbability: number;
 
+  @IsNumber()
+  @Max(1.0)
+  @Min(0.0)
+  @ApiProperty({
+    description: "Probability of leg/trip being cancelled on same day of week (dow) and time in the past.",
+    type: Number,
+    required: true,
+    example: 0.49
+  })
+  cancellationAtDowTimeProbability: number;
```

Comment:
I don't quite get the name. Is this a typo? Should this be `cancellationAtDownTimeProbability`?

Reply:
If yes, this typo appears in other files as well.

Reply:
It is intended this way - for "Day of week" (I admit just looking at this name, that is not clear at all).